robot/client/client.go

```diff
@@ -477,6 +477,10 @@ func (rc *RobotClient) Changed() <-chan bool {
 // Connect will close any existing connection and try to reconnect to the remote.
 func (rc *RobotClient) Connect(ctx context.Context) error {
 	if err := rc.connectWithLock(ctx); err != nil {
+		if strings.Contains(err.Error(), "host appears to be offline") {
```

Comment:
@cheukt 's [comment](https://github.com/viamrobotics/rdk/pull/5466#issuecomment-3528910009) on your other PR may be relevant here, too.